easybuild/easyconfigs/b/Boost/Boost-1.71.0-gompi-2019b.eb

```diff
@@ -0,0 +1,25 @@
+name = 'Boost'
+version = '1.71.0'
+
+homepage = 'http://www.boost.org/'
+description = """Boost provides free peer-reviewed portable C++ source libraries."""
+
+toolchain = {'name': 'gompi', 'version': '2019b'}
+toolchainopts = {'pic': True, 'usempi': True}
+
+source_urls = [SOURCEFORGE_SOURCE]
+sources = ['%%(namelower)s_%s.tar.gz' % '_'.join(version.split('.'))]
+checksums = ['96b34f7468f26a141f6020efb813f1a2f3dfb9797ecf76a7d7cbd843cc95f5bd']
+
+dependencies = [
+    ('bzip2', '1.0.8'),
+    ('zlib', '1.2.11'),
+    ('XZ', '5.2.4'),
+]
+
+configopts = '--without-libraries=python'
```

Comment:
I know this has come up before; do we still want to go with this approach? I've heard this split causes problems for some software that expects a single BOOST_PATH. With python @ core, it wouldn't be to bad to include it here (granted, with the python version suffix).

I don't have a strong opinion either way, but, worth asking.

Reply:
Which Python version would you pick then? Or do we go back to having two `Boost` easyconfigs, one for Py2, one for Py3 (which basically means dragging in Python as a dep everywhere where `Boost` is required).

Related (which complicates things): #8810

Reply:
@Micket would the suggestion made by @boegel in #8810 (symlinking everything from Boost into Boost.Python) solve the problems with BOOST_PATH that you mentioned?

Reply:
@smoors alright that sounds promising.